src/ensembl/src/content/home/components/homepage-app-links/HomepageAppLinks.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+import { isEnvironment, Environment } from 'src/shared/helpers/environment';
+import * as urlFor from 'src/shared/helpers/urlHelper';
+import useOutsideClick from 'src/shared/hooks/useOutsideClick';
+
+import { fetchDataForLastVisitedObjects } from 'src/content/app/browser/browserActions';
+
+import { getEnabledCommittedSpecies } from 'src/content/app/species-selector/state/speciesSelectorSelectors';
+import {
+  getPreviouslyViewedGenomeBrowserObjects,
+  PreviouslyViewedGenomeBrowserObjects
+} from 'src/content/home/homePageSelectors';
+
+import ImageButton from 'src/shared/components/image-button/ImageButton';
+
+import { ReactComponent as BrowserIcon } from 'static/img/launchbar/browser.svg';
+import { ReactComponent as EntityViewerIcon } from 'static/img/launchbar/entity-viewer.svg';
+import { ReactComponent as HomeIcon } from 'static/img/header/home.svg';
+
+import styles from './HomepageAppLinks.scss';
+
+import { RootState } from 'src/store';
+import { Status } from 'src/shared/types/status';
+import { CommittedItem } from 'src/content/app/species-selector/types/species-search';
+
+type Props = {
+  species: CommittedItem[];
+  previouslyViewedGenomeBrowserObjects: PreviouslyViewedGenomeBrowserObjects;
+  fetchDataForLastVisitedObjects: () => void;
```

Comment:
Is `LastVisited` same as using `PreviouslyViewed` ?

Reply:
Probably. The word "last" here actually reflects the reality correctly, because there is only one ensembl object per species that you've seen "last", but there is an indefinite number of objects per species that you've seen "previously".

In any event, this belongs to the very temporary part of the code that will likely be removed when EntityViewer is finally disclosed to the general public. Hopefully we won't then have to query on the home page what objects user has viewed last in which app.

Reply:
Let's leave it as it is if this is just temporary 👍